TODO

```diff
@@ -15,3 +15,4 @@ Drop deprecated Drawable.draw|clean & Graph.draw|clean|getRCommand|makeR*|GetExt
 Drop deprecated GaussianProcess.GIBBS
 Remove FunctionalChaosSobolIndices::summary
 Remove FunctionalChaosResult Function ctor
+Drop deprecated BoxCoxFactory::build methods for 1.21
```

Comment:
I wont be able to merge this, better drop it

Reply:
Done